spec/menu_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu'
+describe Menu do
+  subject(:menu) {described_class.new}
+
+  it 'displays a menu with a list of dishes with prices' do
+    expect(menu.dishes).to eq ({
+      :salad => 12,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)